### PR TITLE
Fix usb2canx2 driver receiving messages

### PR DESCRIPTION
**Pull request by EtcetFelix**
* Fixed various import bugs
* Fixed not being able to receive messages when using usb2canx2 driver

### CODE REVIEW

File: eggs_machina/hw_drivers/transport/can/usb2can_x2.py [hunk 1]
```diff
@@ -16,53 +16,54 @@ def __init__(self, channel: str, baud_rate: int):
...
-                            data=bytes(msg.data)
-                        )
+    def recv(self, can_id: int, is_extended_id: bool, timeout_s: int, *args, **kwargs) -> any:
+        msg = self.bus.recv(timeout=timeout_s) 
+        # for msg in bus:
+        #     if time.time() >= end_time:
+        #         break
+        # print(f"DEBUG, can msg: {msg}")
+        if msg:
+            print("FOUND SOMETHING")
```
Comment (esyywar) [inline eggs_machina/hw_drivers/transport/can/usb2can_x2.py]: intentionally for these to be here or just leftovers from a print debug session? :P

File: eggs_machina/hw_drivers/system/robstride/robstride_types.py [hunk 2]
```diff
@@ -43,12 +43,12 @@ class Robstride_Fault_Frame_Enum(Enum):
 
 @dataclass
 class FeedbackResp:
-    errors: List[Robstride_Fault_Enum]
-    mode: Robstride_Motor_Mode_Enum
-    angle_deg: float
-    velocity_rads: float
-    torque_nm: float
-    temp_c: float
+        errors: List[Robstride_Fault_Enum]
```
Comment (esyywar) [inline eggs_machina/hw_drivers/system/robstride/robstride_types.py]: did these get double indented?

**Review by esyywar — APPROVED** [review]

File: eggs_machina/hw_drivers/transport/can/usb2can_x2.py (hunk 1 above)
Comment (EtcetFelix) [reply eggs_machina/hw_drivers/transport/can/usb2can_x2.py]: ya leftover, my b

File: eggs_machina/hw_drivers/system/robstride/robstride_types.py (hunk 2 above)
Comment (EtcetFelix) [reply eggs_machina/hw_drivers/system/robstride/robstride_types.py]: looks like it, fixing

**Pull request merged**